lib/SPIRV/SPIRVLowerSPIRBlocks.cpp

```diff
@@ -226,7 +226,7 @@ class SPIRVLowerSPIRBlocks : public ModulePass {
             assert(CI->getArgOperand(0) == CallBlkBind);
             Changed |= lowerGetBlockInvoke(CI, cast<Function>(InvF));
           } else if (Name == SPIR_INTRINSIC_GET_BLOCK_CONTEXT) {
-            assert(CI->getArgOperand(0) == CallBlkBind);
+            assert(Ctx && CI->getArgOperand(0) == CallBlkBind);
```

Comment:
I suggest to place an assertion for `Ctx` right after call to `getBlockInvokeFuncAndContext` at line 210: I think it is better to check result of the function right after call to it and `Ctx` is also used on line 234, where it is not covered by the proposed change